app/pages/cif/projects.tsx

```diff
@@ -67,10 +70,18 @@ export const ProjectsQuery = graphql`
         }
       }
     }
-    allCifUsers {
+    allProjectManagers(
```

Comment:
I don't totally follow what was previously going on here--why was the query using users instead of managers?

Reply:
I'm not sure either; When I was trying to search for a name, I found my name there too while I'm not a manager. but since AC only requires primary managers to show up in the search list and that's why I changed it to allProjectManagers.

Reply:
Actually, late-breaking thought, it might be because we don't have a way (as far as I know) for users to set up managers. That query comes back with April Ludgate etc. because we set her up as a manager in the dev data, but we might get no one back in test/prod. @pbastia, thoughts?

Reply:
Something like this will do what you want:
```graphql
query MyQuery {
  allCifUsers(
    filter: {projectManagersByCifUserId: {some: {projectManagerLabelByProjectManagerLabelId: {label: {includesInsensitive: "primary"}}}}}
  ) {
    edges {
      node {
        fullName
      }
    }
  }
}
```